geth/profiling/profiling.go

```diff
@@ -0,0 +1,55 @@
+package profiling
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"runtime/pprof"
+)
+
+const (
+	CPUProfilingFilename = "status_cpu.prof"
+	MemProfilingFilename = "status_mem.prof"
```

Comment:
As they are exported, can we change their name to `CPUFilename` and `MemFilename` so that their usage is more elegant, e.g.: `profiling.CPUFilename`.